src/transformers/models/metaclip_2/configuration_metaclip_2.py

```diff
@@ -4,7 +4,6 @@
 #             the file from the modular. If any change should be done, please apply the change to the
 #                          modular_metaclip_2.py file directly. One of our CI enforces this.
 #                🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨
-
```

Comment:
added to fix https://app.circleci.com/pipelines/github/huggingface/transformers/150805/workflows/8ab65b66-dc7a-42a4-9b24-ad138a35dc71/jobs/1985990